src/clr-angular/accordion/stepper/models/stepper.model.ts

```diff
@@ -19,7 +21,9 @@ export class StepperModel extends AccordionModel {
 
   updatePanelOrder(ids: string[]) {
     super.updatePanelOrder(ids);
-    this.openFirstPanel();
+    if (this.stepperModelInitialize === false) {
```

Comment:
Can we have a test to assert this behavior?

Reply:
Sure, but it's a bit hard to test private methods and properties. 

Updated ...